### PR TITLE
fix(TDP-3984): no feedback in share dialog

### PR DESCRIPTION
**Pull request by ncomont**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-3984

**Please check if the PR fulfills these requirements**
- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [ ] The new code does not introduce new technical issues (sonar / eslint)
- [ ] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [ ] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/746/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/746/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/747/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/747/)

File: dataprep-webapp/src/app/components/folder/selection/_folder-selection.scss [hunk 1]
```diff
@@ -29,4 +29,12 @@
     margin-top: 10px;
     padding: 2px;
   }
+
+  .loader-container {
```
Comment (charlesnguyen) [inline dataprep-webapp/src/app/components/folder/selection/_folder-selection.scss]: this class is duplicated everywhere (ex: in _action-suggestions.scss, _column-profile.scss, _folder-selection.scss). it should be put in the "loader" component or declare as a "global" class i guess.

File: dataprep-webapp/src/app/components/loader/_loader.scss [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
-		}
+	.loading-msg {
+		text-align: center;
+		background: transparent;
+		color: $pale-blue;
+		opacity: 0.8;
+
+		.wait {
+			&-icon {
+				width: 16px;
```
Comment (charlesnguyen) [inline dataprep-webapp/src/app/components/loader/_loader.scss]: why width and height are added?

**Pull request merged**